src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -517,6 +521,8 @@ class TrialWaveFunction
   ///a list of WaveFunctionComponents constituting many-body wave functions
   std::vector<std::unique_ptr<WaveFunctionComponent>> Z;
 
+  /// For now, TrialWaveFunction will own the wrapper.
+  TWFPrototype twf_prototype;
```

Comment:
Your test case shows it is owned outside TWF. Do we still need this?

Reply:
I had always envisioned this as being owned by TrialWaveFunction, but based on this example...  what do you think of the following:  What if the force observable owns the wrapper for now?  That would make initialization unambiguous...  it would be initialized by the object that needs it.

Reply:
I forgot if I have mentioned this in any comments. TWFPrototype is still not very informative.
TWFStructureMetaData can be better. We can discuss the name here but any renaming can happen after all the pieces land.

Reply:
> I had always envisioned this as being owned by TrialWaveFunction, but based on this example... what do you think of the following: What if the force observable owns the wrapper for now? That would make initialization unambiguous... it would be initialized by the object that needs it.

If calls are only from force estimator, the force estimator can own it and call the initialization in its constructor.

Reply:
You had not, but I agree...  TWFPrototype was a placeholder name.  TWFStructureMetaData can be fine, TWFDerivWrapper might be descriptive.  I'm totally open to suggestions.  Should just be a sed command once we decide.  I'd probably kick the can down the road until we have a complete working code, but I don't feel particularly strongly one way or the other.

Reply:
> 

We'll do it that way for now then.  If it looks like this can/will be generalized to other derivatives, then we can change the ownership/scope then.